include/dqrobotics/interfaces/coppeliasim/DQ_CoppeliaSimInterfaceZMQ.h

```diff
@@ -204,6 +193,19 @@ class DQ_CoppeliaSimInterfaceZMQ: public DQ_CoppeliaSimInterface
         if (static_cast<std::size_t>(v1.size()) != static_cast<std::size_t>(v2.size()))
             throw std::runtime_error(error_message);
     }
+
+    std::string _get_object_name(const int& handle);
+
+    template <typename T, typename = typename std::enable_if<std::is_arithmetic<T>::value>::type>
+    std::vector<std::string> _get_object_names(const std::vector<T>& handles)
```

Comment:
Thanks, @juanjqo. We're just missing the docs for this template and the `_check_sizes()` (or similar name). That one might have slipped through.

Reply:
@mmmarinho fixed!